Assets/Python/Barbs.py

```diff
@@ -70,12 +70,12 @@
 (900, (25, 23), iNative, 'Chan Chan', 2, con.iArcher, 2),		# Chan Chan
 (900, (69, 52), iIndependent, 'Kyiv', 2, con.iLongbowman, 2),		# Kiev
 (990, (49, 56), iCeltia, '&#193;th Cliath', 1, -1, -1),			# Dublin
-(1000, (61, 63), iIndependent2, 'Nidaros', 1, con.iVikingBerserker, 1),	# Trondheim
-(1000, (71, 17), iNative, 'Quelimane', 1, con.iZuluImpi, 1),		# Quelimane
-(1100, (71, 20), iNative, 'Mombasa', 1, con.iZuluImpi, 1),		# Mombasa
+(1000, (61, 63), iIndependent2, 'Nidaros', 1, con.iVikingHuskarl, 1),	# Trondheim
+(1000, (71, 17), iNative, 'Quelimane', 1, con.iBarbarianImpi, 1),		# Quelimane

```

Comment:
Actually I'd prefer to retain the original civs in those tags, instead of Barbarian, even if they are not in the game.